Overview.src.html

```diff
@@ -442,10 +442,38 @@ <h4 id=terminology-headers>Headers</h4>
 
 <h4>Bodies</h4>
 
-<p>A <dfn title=concept-body>body</dfn> is a byte stream. It has an associated
-<dfn title=concept-body-transmitted>transmitted</dfn> which is an integer and initially 0,
-<dfn title=concept-body-length>length</dfn> which is an integer and initially 0, and
-<dfn title=concept-body-error-flag>error flag</dfn> which is initially unset.
+<p>A <dfn title=concept-body>body</dfn> consists of a
+<code title=concept-ReadableStream>ReadableStream</code> object and accompanying data.
+
+<p>A <span title=concept-body>body</span> has an associated
+<dfn title=concept-body-stream>stream</dfn> (a
+<code title=concept-ReadableStream>ReadableStream</code> object).
+
+<p class="XXX"><a href="https://github.com/whatwg/streams/issues/379">This might become a
+<code>ReadableByteStream</code> object</a>. While the type might change, the behavior specified will
+be equivalent since the hypothetical <code title>ReadableByteStream</code> object will have the same
+members as the <code title=concept-ReadableStream>ReadableStream</code> object has today.
+
+<p>A <span title=concept-body>body</span> has an associated
+<dfn title=concept-body-transmitted>transmitted</dfn> (an integer). Unless stated otherwise it is 0.
+
+<p>A <span title=concept-body>body</span> has an associated
+<dfn title=concept-body-total-bytes>total bytes</dfn> (an integer). Unless stated otherwise it is 0.
```

Comment:
maybe keep the concept-body-length ID to not break links? @annevk would need to make the call on that.